tests/restarting/to_7.3.0_until_7.3.39/CycleTestRestart-1.toml

```diff
@@ -2,9 +2,11 @@
 maxTLogVersion = 6
 disableTss = true
 disableHostname = true
-encryptModes = ['disabled']
-tenantModes=['disabled']
```

Comment:
why removing these two?

Reply:
I see. I just deleted the folder of to_7.2.0_until_7.3.0 and created a new folder of to_7.3.0_until_7.3.39. Therefore, the git diff compares the two files under two different folders.

Do we want to add   `to_7.2.0_until_7.3.0` back?

Reply:
I see. No need to add `to_7.2.0_until_7.3.0` back, since we won't be testing with 7.2 binaries.